Program Resources/Documentation Enhancement Grants README.md

```diff
@@ -0,0 +1,47 @@
+
+# Documentation Enhancement Grants
+
+## About
+Do you have a great idea to enhance existing Filecoin documentation or to create new resources? Consider applying for a Documentation Enhancement Grant! These grants are intended for anyone interested in contributing to the enhancement of Filecoin documentation. 
+
+Grant proposals should fit within the following focus areas:
+
+- **Tutorial Creation**: Create tutorials and walkthroughs demonstrating how to use Filecoin APIs, SDKs, and tools for common use cases (i.e. storing and retrieving data, managing storage deals, interacting with smart contracts, etc.)
+
+- **Comprehensive Developer Guides**: Develop detailed guides for developers covering various aspects of integrating Filecoin into their projects, including setup instructions, API usage, and troubleshooting tips.
+
+- **Interactive Learning Resources**: Develop interactive learning resources such as code playgrounds or interactive demos/videos that allow developers to experiment with Filecoin APIs and tools.
+
+## Program Qualifications
+Filecoin Documentation Enhancement Grants must meet the following criteria:
+
+- You have a clear plan to improve existing documentation or create new resources related to Filecoin.
+- You can provide a clear and straightforward description of the documentation enhancement project..
+- You can complete this work within 3 months from signing a grant agreement with Filecoin Foundation.
+- You agree to open-source the enhanced documentation, via a suitable license.
+- You agree to provide regular updates on the progress of your documentation enhancement project. This can be completed in Github.
+- If selected, grant applicants will need to provide additional information during the contracting and invoicing process.
+
+## How to Apply and Review Timeline
+To submit a proposal, [create a new issue](https://github.com/filecoin-project/devgrants/issues/new/choose) using the “Documentation Enhancement Grants" proposal template.
+
+We aim to notify Documentation Enhancement Grant applicants of the review decision by the end of the month following the application submission date.
+
+- For example, for applications submitted between March 1 and March 31, applicants will be notified by April 30.
+
+Our team may be in touch with questions in the proposal comment section in Github. **Please note that no work should begin without a signed contract.** 
+
+## Award
+Grants will be paid out once the following steps are completed:  
+- A grant agreement is signed by both parties
+- The outlined project work is complete
+  - As part of their final submission, all grantees should demonstrate successful usage by members of their intended audience with the intended audience as part of the project completion report.
```

Comment:
Rephrase to "As part of the final submission"

Reply:
Great catch@